modules/quickcreds

```diff
@@ -1,148 +1,222 @@
 #!/bin/bash /usr/lib/turtle/turtle_module
-VERSION="1.3"
-DESCRIPTION="Snagging creds from locked machines --Mubix, Room362.com. Implements responder attack and saves creds to numbered directories in /root/loot. LED will blink while QuickCreds is running. Upon capture of NTLM hash the amber LED will light solid. Author: Hak5Darren. Credit: Mubix."
+VERSION="1.3-KoalaV2"
+DESCRIPTION="Snagging creds from locked machines --Mubix, Room362.com. Implements responder attack and saves creds to numbered directories in /root/loot. LED will blink while QuickCreds is running. Upon capture of NTLM hash the LED will light solid. Author: Hak5Darren. Credit: Mubix."
 CONF=/tmp/QuickCreds.form
+# Original file:
+# https://github.com/hak5/lanturtle-modules/blob/gh-pages/modules/quickcreds
 
+# All "dialog" functionality is related to the Turtle "GUI"
 : ${DIALOG_OK=0}
 : ${DIALOG_CANCEL=1}
 : ${DIALOG_HELP=2}
 : ${DIALOG_EXTRA=3}
 : ${DIALOG_ITEM_HELP=4}
 : ${DIALOG_ESC=255}
 
-function configure {
-#  dialog --title "QuickCreds" --msgbox "\n\
-# Dependencies will be installed. An Internet connection is required.\n\
-# Upon configuration creds will be saved to /root/loot/ on boot." 9 72
-
-dialog --title "QuickCreds" \
---yesno "\nInstall dependencies and configure QuickCreds?\n\
-An Internet connection is required for installation.\n" 8 60
-response=$?
-case $response in
-   0) ;;
-   1) exit ;;
-   255) exit ;;
-esac
-
-# Check to see if QuickCreds has already been configured
-if [[ -d /root/loot || -s /root/loot/responder.log ]];
-then
-  dialog --title "QuickCreds" --msgbox "\nThe QuickCreds module is already configured.\n\
-Creds are saved to /root/loot.\nEnable this module to attack on boot." 9 72
-  exit 1
+# Declare some additional variables
+LOOTDIR="/root/loot/quickcreds/"
+RESPLOG=$LOOTDIR"responder.log"
+RESPROOT="/sd"
+RESPTEMPDL="/tmp/ResponderDownload/"
+RESPURL="https://github.com/lgandx/Responder/archive/master.zip"
+RESPLOGDIR=$RESPROOT/"Responder/logs"
+SLEEPTIMER="1" # Don't set this to less than 1 (one) if using a LAN Turtle with a yellow LED
+
+# Check what type of controllable LED that is available, either yellow or orange/amber
+LEDFILE=/sys/class/leds/turtle\:yellow\:system/brightness
+if [[ ! -f "$LEDFILE" ]]; then
+    # The files to control the yellow LED is absent, use the orange/amber variant
+    LEDYELLOW=0
+else
+    LEDYELLOW=1
```

Comment:
This check is not required. The change in LED path comes from a firmware change. It is the same path for all devices on the latest firmware. This can be removed, just use the new path.